src/gui/virtualstudio.cpp

```diff
@@ -756,7 +756,7 @@ void VirtualStudio::completeConnection()
     }
 
     m_jackTripRunning = true;
-    m_connectionState = QStringLiteral("Connecting...");
+    m_connectionState = QStringLiteral("Preparing audio...");
```

Comment:
If we want to preserve the functionality of `sendHeartbeat()` not being invoked after this line, we might need to change [this check using m_connectionState](https://github.com/jacktrip/jacktrip/blob/main/src/gui/virtualstudio.cpp#L1103-L1105) to
```
if (m_device != nullptr && m_connectionState != "Connecting..." && m_connectionState != "Preparing audio...") {
```